Examples/example_python.py

```diff
@@ -1,6 +1,6 @@
-import src.parastell as ps
 import numpy as np
 
+import parastell as ps
```

Comment:
As the file tree is set up, assuming the repository base directory is what's on the user's `PYTHONPATH`, the proper import syntax would be
```python
import parastell.parastell as ps
```
This would apply to the other example script as well, of course.